src/pushsource/_impl/model/erratum.py

```diff
@@ -265,6 +266,34 @@ class ErratumPushItem(PushItem):
     :type: list[ErratumPackageCollection]
     """
 
+    container_list = attr.ib(
+        type=list, default=attr.Factory(frozenlist), converter=frozenlist
+    )
+    """A list of container images associated with the advisory.
+
+    Example of the list:
```

Comment:
This won't render correctly in the docs, you'll have to do something like:

- blank space after this line
- put the example in an indented `.. code-block:: python`

You can find examples if you grep for `code-block::`.